src/data/markdown/docs/03 cloud/03 Manage/04 Environment variables.md

```diff
@@ -0,0 +1,95 @@
+---
+title: 'Environment variables'
+excerpt: 'How to use environment variables in the cloud'
+---
+
+Environment Variables are key-value pairs configured at the organization level. All values are encrypted before being stored in our database and remain encrypted until they are needed for test-runs. You reference environment variables within a test-script and the variables are interpolated on the server that runs the Test-Run. Environment Variables can be used to store sensitive (and non-sensitive) information that you want to reference in your test-scripts. Managing environment variables in one central place reduces the data exposure and ensures that the changes take effect in all test-runs that use these variables.
+
+> Changes to an Environment Variable won't apply to already started Test-Runs, they only apply to new upcoming Test-Runs.
+
+## Managing environment variables
+
+Environment Variables can be managed in Organization Settings: Select your profile icon, then **Manage > Environment variables**. Permission to create, reveal, modify and delete environment variables is restricted to Owners and Admins of the organization. 
+
+![k6 Environment Variable](./images/envvars/environment-variables.png)
+
+#### Naming environment variables
+
+The following rules must apply for naming an environment variable:
+- Names can only contain alphanumeric characters (`[a-z]`, `[A-Z]`, `[0-9]`) or underscores (`_`). Spaces are not allowed.
+- Names must not start with the `K6_CLOUD` prefix.
+- Names must not start with a number.
+- Names are not case-sensitive.
+- Names must be unique.
+
+#### Declaring a new environment variable
+
+Enter the Name for your Environment Variable. Then, enter the respective Value. Optionally, you can also add a brief description then click **submit**. 
+
+![k6 Environment Variable](./images/envvars/create-environment-variable.png)
+
+### Accessing environment variables
```

Comment:
Should this be H2? I think it does not belong to `Managing environment variables` section